src/DynamoUtilities/DynamoException.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
I think these are just default ones, please remove the unused ones

Reply:
Does SQ generate a smell for keeping the default using? That's even worse than FxCop :)

Reply:
@mmisol They do :)